numba/ir_utils.py

```diff
@@ -107,12 +112,12 @@ def mk_range_block(typemap, size_var, calltypes, scope, loc):
         phi_assign, jump_header]
     return range_block
 
-def _get_range_func_typ():
-    """get type variable for range() from builtin registry"""
+def get_global_func_typ(func):
+    """get type variable for func() from builtin registry"""
     for (k,v) in typing.templates.builtin_registry.globals:
-        if k==range:
+        if k==func:
             return v
-    raise RuntimeError("range type not found")
+    raise RuntimeError("func type not found"+str(func))
```

Comment:
suggestion: using string formatting is preferred.  Either the new-style (`.format`) or old-style (`%`) will do.  This will then be `"func type not found {}".format(func)`